src/System.Memory/tests/ReadOnlySpan/ToString.cs

```diff
@@ -43,11 +43,56 @@ public static void ToStringChar_Empty()
         }
 
         [Fact]
-        public static unsafe void ToStringForSpanOfString()
+        public static void ToStringForSpanOfString()
         {
             string[] a = { "a", "b", "c" };
             var span = new ReadOnlySpan<string>(a);
             Assert.Equal("System.ReadOnlySpan<String>[3]", span.ToString());
         }
+
+        // This test is only relevant for portable span
+#if FEATURE_PORTABLE_SPAN
+        [Fact]
+#endif
```

Comment:
Is this the convention used in this project?  Elsewhere in corefx we've used SkipOnTargetFramework, e.g. https://github.com/dotnet/corefx/pull/27170/files#diff-f8d6f156088e92893813570d9608526eR63

Reply:
I am not sure if NetFramework is the only one TargetFrameworkMoniker where we would skip this test.

Reply:
Ok.

Reply:
But given [this](https://github.com/dotnet/corefx/blob/7059f8b316723f5db64e4b1d28e2b1e8b137ab1e/Documentation/project-docs/developer-guide.md#skipontargetframeworkattribute), I will change it: